Overview.bs

```diff
@@ -91,22 +91,99 @@ Introduction {#intro}
 slow networks, very large fonts, or complex subsetting requirements currently preclude their use. For
 example, even using WOFF 2 [[WOFF2]], fonts for CJK languages are too large to be practical.
 
+There are two different methods which can be used to incrementally transfer fonts.
+
+Patch Subset {#patch-subset}
+----------------------------
+
+The first method,
+<a href="patch-incxfer">Patch Subset</a>, uses a server which can generate binary updates to an existing font file.
+
+Range Request {#range-request}
+------------------------------
+
+The second
+method, <a href="#range-request-incxfer">Range Request</a>, is a very simple method of incremental transfer, and has no server-side requirements (other than the server should be able to respond to byte-based range requests). The browser simply makes range requests to the server for the specific bytes in the font file that it needs. In order to know which bytes are necessary, the browser makes one initial special request for the beginning of the file.
+
+In order for the range request method to be as effective as possible, the font file itself should be internally arranged in a particular way, in order to decrease the number of requests the browser needs to make. Therefore, it is expected that web developers wishing to use the range request method will use font files that have had their contents already arranged optimally.
+
+This method was modelled after video playback on the web, where seeking in a video causes the browser to send a range request to the server.
+
+Technical Motivation: Evaluation Report {#evaluation-report}
+------------------------------------------------------------
+
 See the Progressive Font Enrichment: Evaluation Report [[PFE-report]] for the investigation which led
 to this specification.
 
-There are two different methods which can be used to incrementally transfer fonts. The first method,
-Patch Subset, uses a backend server which can generate binary patches to an existing font. The second
-method, Range Request, utilizes HTTP range requests to load only the parts of the original font that
-are needed.
-
 The evaluation report found that patch subset was generally more efficient in terms of overall
 performance and transferred bytes than range request. However, Range Request is simpler to deploy
-for many uses cases while still providing material improvments to loading performance for large fonts
-so it is included in this specification as an alternative method.
+for many uses cases while still providing material improvments to loading performance for large fonts.
+
+Opt-In Mechanism {#opt-in}
+==========================
+
+<em>This section is general to both IFT methods.</em>
+
+The collection of IFT technologies utilizes a single shared opt-in mechanism. Each method does not use its own opt-in mechanism; instead, the webpage opts into the IFT technologies as a whole, and the browser and server negotiate to decide which specific method will be employed.
+
```

Comment:
This section warrants additional discussions and may need to be revised. The opt-in mechanism is controlled by authors, including their preference of IFT method.